static/templates/settings/notification-settings.handlebars

```diff
@@ -67,6 +67,19 @@
 
       <div class="input-group">
         <label class="checkbox">
+          <input type="checkbox" name="hide_private_message_desktop_notifications" id="hide_private_message_desktop_notifications"
+                 {{#if page_params.hide_private_message_desktop_notifications}}
+                 checked="checked"
+                 {{/if}} />
+          <span></span>
+        </label>
+        <label for="hide_private_message_desktop_notifications" class="inline-block">
+          {{t "Hide Message" }}
```

Comment:
This should have a longer/clearer explanation, and possibly the meaning of the flag flipped (making it default-true):
"Include content of private messages in desktop notifications."